python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -271,16 +271,24 @@ def _add_filters_to_query(self, query: SqlAlchemyQuery, filters: RunsFilter) ->
             query = query.where(RunsTable.c.snapshot_id == filters.snapshot_id)
 
         if filters.updated_after:
-            query = query.where(RunsTable.c.update_timestamp > filters.updated_after)
+            query = query.where(
+                RunsTable.c.update_timestamp > filters.updated_after.replace(tzinfo=None)
```

Comment:
this is a pre-existing bug that this PR surfaced, if you didn't pass in a naive datetime we weren't passing it in correctly